src/script/components/Conversation/Conversation.tsx

```diff
@@ -118,8 +121,11 @@ export const Conversation = ({
 
   const [isMsgElementsFocusable, setMsgElementsFocusable] = useState(true);
 
+  const [messagesListRerenderKey, rerenderMessageList] = useComponentRerenderKey('messages-list');
```

Comment:
This seems like a non-trivial hook. Can you add comment to explain why this is needed and what's the purpose of it?

Reply:
This is the way to trigger the re-render of `MessageList`, by changing its `key` (KUDOs to @PatrykBuniX btw). Should I add a comment to the code?

Reply:
That's clever solution actually. I like it!!

But, because it's clever, it does call for a code-comment :)

Reply:
done